tests/main/interfaces-calendar-service/task.yaml

```diff
@@ -7,7 +7,8 @@ summary: Ensure that the calendar-service interface works
 # https://github.com/snapcore/snapd/pull/7230 is landed
 # ubuntu-19.10: test-snapd-eds is incompatible with eds version shipped with the distro
 # arch-linux: test-snapd-eds is incompatible with eds version shipped with the distro
-systems: [-ubuntu-core-*, -ubuntu-14.04-*, -amazon-*, -centos-*, -opensuse-tumbleweed-*, -ubuntu-19.10-*, -arch-linux-*, -debian-sid-*]
+# fedora-31: test-snapd-eds is incompatible with eds version shipped with the distro
```

Comment:
should we change the interface provided by the system to address this? Seeing how all new distros are unsupported makes me wonder if the interface slot needs to be versioned.

Reply:
AFAIK there's nothing we can do in snapd. The client library expects the evolution-data-server to register itself under a specific name, which also encodes the API version. The newer versions of eds have a different API version.

Reply:
Nothing to do in this PR (apart from disabling fc31), but I agree we should think about versioning this interface, and maybe filter out the old version of this interface on systems known to require the new one.
I feel we will some more of those kind of API incompatibilities in the future.